packages/migrations/src/batched-migrations/batched-migration-runner.ts

```diff
@@ -177,6 +177,7 @@ export class BatchedMigrationRunner {
       }
 
       if (error) {
+        logger.error('Error running the batched migration job.', error);
```

Comment:
The nittiest of nits (this more closely matches our other logs):

```suggestion
        logger.error('Error running batched migration job', error);
```